ui/features/landing/landing.controller.js

```diff
@@ -14,10 +14,11 @@ export default class LandingController {
     this.$rootScope = $rootScope;
     this.model = null;
     this.neighbors = [];
-    var i = 0;
+    var color = {'subClassOf':'#FF4646','~subClassOf':'#992A2A','isDefinedBy':'#00FF00','~isDefinedBy':'#005B00','equivalentClass':'#5BCBB6','~equivalentClass':'#2D655B','sameAs':'#774177','subPropertyOf':'#FFC04C','!subPropertyOf':'#A47B30','equivalentProperty':'#6666FF','~equivalentProperty':'#3D3D99','property':'#FF7032','~property':'#CC3D00','type':'#B2FFFF','~type':'#EDD95D','operand':'#000000'};
     var duration = 750;
```

Comment:
I was thinking about it following that good formatting, but decided against it for this variable because it's so big and it's just colors that I didn't think would qualify as "code" for someone to go in and change. But I'll change it now.